pychromecast/controllers/spotify.py

```diff
@@ -20,12 +20,13 @@ class SpotifyController(BaseController):
     # pylint: disable=useless-super-delegation
     # The pylint rule useless-super-delegation doesn't realize
     # we are setting default values here.
-    def __init__(self, access_token):
+    def __init__(self, access_token, expires=3600):
```

Comment:
I don't think that we should default the `expires`. It should be good for users to be aware why they need to pass this in so it's more clear when things stop working.

Reply:
I removed the default value and added a simple check for the parameters